Add uint16 to supported dtypes and regenerate edge.yaml

tarun292 opened the pull request:
Summary: Adding `uint16` support and regenerating `edge.yaml` as we now support `uint16` in the ExecuTorch stack.

Differential Revision: D68241997

zonglinpeng reviewed (approved):
I can confirm it works for G3 permute thanks!

tarun292 committed:
Add uint16 to supported dtypes and regenerate edge.yaml (#7687)

Summary:

Adding `uint16` support and regenerating `edge.yaml` as we now support `uint16` in the ExecuTorch stack.

Reviewed By: zonglinpeng

Differential Revision: D68241997
